fitnessllm_dataplatform/stream/strava/services/silver_etl_interface.py

```
@@ -0,0 +1,45 @@
+
+
+
+        for query in list_of_queries:
+            target_destination = f"{self.ENV}_silver_{self.data_source.value.lower()}.{query.split('.')[0]}"
+            query_path = pathlib.Path(path, query)
+
+            out = get_transaction_insert_query(target_table=target_destination,
+                                               query_path=query_path,
+                                               parameters=parameters)
+
+            result = self.client.query(out).result()
```

Ruff (0.8.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)